examples/client/lwm2mclient.c

```diff
@@ -286,6 +324,12 @@ void * lwm2m_connect_server(uint16_t secObjInstID,
 }
 #endif
 
+/**
+ * @brief Closes the connection.
```

Comment:
Needs more details. Explaining what happens e.g. WITH_TINYDTLS is defined /not defined?What does lwm2m_free do? etc.